src/main/java/com/softserve/bookstore/controllers/BookController.java

```diff
@@ -67,7 +66,7 @@ public ResponseEntity<ErrorResponse> handleSQlEx(SQLException msg) {
     }
 
     @ExceptionHandler({IOException.class})
-    public ResponseEntity<ErrorResponse> handleIoEx(SQLException msg) {
+    public ResponseEntity<ErrorResponse> handleIoEx(IOException msg) {
```

Comment:
same thing as in previous comment

also naming for variable
message is just fine, do not shorten names  unless  it is really necessary

otherwise it will get harder to read, as mentioned before more on that in Clean Code book,  I think it was chapter 1